lib/services/android-project-service.ts

```diff
@@ -298,6 +298,10 @@ export class AndroidProjectService extends projectServiceBaseLib.PlatformProject
 		return this.$fs.exists(path.join(this.platformData.appDestinationDirectoryPath, constants.APP_FOLDER_NAME));
 	}
 
+	public isPlatformBuilt(projectRoot: string): IFuture<boolean> {
+		return this.$fs.exists(path.join(projectRoot, "build", "outputs", "apk"));
```

Comment:
Is this enough? Shouldn't you check for existing file with extensions .apk inside this directory? What will happen in case it is empty?